src/main/java/pageobject/TestSessionPage.java

```diff
@@ -0,0 +1,250 @@
+	
+	/* Xpath for Delete the 'TestSession' */
+	By BtnDeleteTestSession = By.xpath("(//SPAN[@class='f-title f-underline'][text()='Delete'])[1]");
+	By PopUp_DeleteTestSeeion = By.xpath("//button[@class='f-button f-bg-1 f-bold f-button f-c f-color-w f-h-40 f-lh-40 f-pad-1-h f-size-14 f-w-175 pure-input-1'][text()='Delete']");
+	By PopUp_DeleteSuccess = By.xpath("(//I[@class='icon icon-close-4 f-abs-all'])[1]");
+	
+	
+	
+	/*public void linkDevTools() throws Exception {
+		try {
+			clickElement(DevTools);
+			//driver.findElement(DevTools).click();
+			System.out.println("DevTools link clicked");
+		}
+
+		catch (Exception e) {
+			System.out.println("DevTools link not found");
+			throw (e);
+		}
+	}*/
+	
+	public void linkTestSession() throws Exception {
+		try {
+			System.out.println("Inside Link_Test_Session TestSession Page Method");
+			Thread.sleep(3000);
+			//ajaxScroll(TestSessions);	
+			JavascriptExecutor js = (JavascriptExecutor) driver;
+			js.executeScript("window.scrollBy(0,500)");			
+			Thread.sleep(2000);
+			clickElement(TestSessions);			
+			System.out.println("TestSessions link clicked");
+		}catch (Exception e) {
+			System.out.println("TestSessions link not found");
+			throw (e);
+		}
+	}
+	
+		
+	public void btn_AddTestSession() throws Exception {
+		try {
+			Thread.sleep(2000);
+			clickElement(AddTestSession);			
+			System.out.println("AddTestSession button clicked");
+		}catch (Exception e) {
+			System.out.println("AddTestSession button not found");
+			throw (e);
+		}
+	}
+	
+	
+	public void dropDown_SelectCampaign() throws Exception {
+		try {
+			Thread.sleep(3000);
+			clickElement(SelectCampaign);
+			//driver.findElement(SelectCampaign).click();
+			System.out.println("SelectCampaign drop-down open");
+			Thread.sleep(3000);
+			clickElement(selectCampaignValue);
+			System.out.println("Campaign Value selected");
+		}catch (Exception e) {
+			System.out.println("SelectCampaign drop-down not found");
+			throw (e);
+	}
+  }
+	
+	
+	public void select_ExpirationDate() throws Exception {
+		try {
+			Thread.sleep(2000);
+			clickElement(ExpirationDate);
+			System.out.println("ExpirationDate selected");
+			clickElement(todayDate);
+			System.out.println("todayDate selected");
+		}catch (Exception e) {
+			System.out.println("ExpirationDate not selected");
+			throw (e);
+	}
+  }
+	
+	public void select_Numbers() throws Exception {
+		try {
+			Thread.sleep(2000);
+			clickElement(Numbers);
+			System.out.println("Numbers selected");
+			
+		}catch (Exception e) {
+			System.out.println("Numbers not selected");
+			throw (e);
+	}
+  }
+	
+	public void enter_SessionData() {		
+		try {			
+			
+			Thread.sleep(2000);
+			clickElement(EnterSessionData);
+			FileReader reader = new FileReader("Binaries/SampleJson.json");
+			
+			//sendKeysWithAjax(EnterSessionData, "{}");
+			//System.out.println("Session data Entered............");
+			JSONParser jsonpaser = new JSONParser();
```

Comment:
Create common function in BaseAction.class and then use it.

Reply:
Working on it.